drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -153,46 +154,47 @@ def fillChecksumMap(checksumFilePath):
 
 # creates a file containing checksums and paths for files contained in the passed path using the global checksum dictionary
 def createChecksumFileForFolder(incomingPath, folderPath):
-    relativePath = os.path.relpath(incomingPath, folderPath)
-    print incomingPath
-    print folderPath
-    print relativePath
+    relativePath = os.path.relpath(folderPath, incomingPath)
 
     pathEnd = os.path.basename(os.path.normpath(folderPath))
     checksumFile = os.path.join(folderPath, pathEnd+'.sha256sum')
+    print checksumFile
+    print os.path.exists(folderPath)
     with open(checksumFile, 'w') as f:
-        for key, value in dataMap.items():
+        for key, value in checksumMap.items():
             if key.startswith(relativePath):
                 f.write(value+' *'+key)
 
-def prepareDataFolder(incomingPath, currentPath, targetPath, dataObject):
+def prepareDataFolder(incomingPath, currentPath, targetPath, dataObject, suffix):
     name = dataObject.getName()
-    src = os.path.join(currentPath, name)
+    rel = dataObject.getRelativePath()
+    print rel
```

Comment:
Clean Code best practices: declarative variable name. Also, this print statement would difficult to match to the ETL script, when we go through the openbis logs.